packages/frontend/src/lib/BootcEmptyScreen.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+import EmptyScreen from './upstream/EmptyScreen.svelte';
+import BootcIcon from './BootcIcon.svelte';
+</script>
+
+<EmptyScreen
+  icon="{BootcIcon}"
+  title="No past bootc build available"
```

Comment:
To be more consistent with PD just 'No bootc builds' or 'No bootable container builds'?